ui/src/main/webapp/lib/ddf-sources/index.js

```diff
@@ -25,7 +24,7 @@ const CSW = {
   `,
   mutation: gql`
     mutation SaveCswSource($config: CswSourceConfiguration!){
```

Comment:
Yes, I will rename that. The update source methods are not there because they are not being used by the front end for the moment but the functionalities exist.